PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -1,23 +1,92 @@
 <template>
 	<v-container fluid>
+    <v-row justify="center">
+  <v-col class="d-flex" cols="6" sm="6">
+  <v-text-field
+      v-model="qrText"
+      clearable
+      counter
+      filled
+      label="Type some text here"
+      @input="generateQrCode"
+    ></v-text-field>
+    </v-col></v-row>
+
+    <v-row justify="center">
+<div class="justify-center" v-if="qrSrc">
+  <img class="preview" :src="qrSrc"> </div>
+    </v-row>
+
     <v-row>
-QR Code Page
+    <v-btn
+      color="accent"
+      :disabled="!qrSrc"
+      @click="reset"
+    >
+    Reset
```

Comment:
We should anchor the "reset" and "open" buttons to the bottom of the page to be consistent with our UI. I'd be ok not doing that if these buttons will change in the future. 
![image](https://user-images.githubusercontent.com/67079030/102700106-29047500-4210-11eb-8189-555ea5ec05f1.png)

Reply:
These buttons will be changed later to Download & Send, once the QR code generator is connected to the ID generator.